src/explore-education-statistics-common/src/modules/find-statistics/hooks/useTableQuery.ts

```diff
@@ -30,13 +31,16 @@ export default function useTableQuery(
       return undefined;
     }
 
-    const queryKey = JSON.stringify(query);
+    const queryKey = JSON.stringify({
+      ...query,
+      dataLastPublished,
+    });
 
     let response = await storageService
       .get<TableDataResponse>(queryKey)
       .catch(() => null);
 
-    if (!response) {
+    if (response == null) {
```

Comment:
Would keep this check as `!response`. Using `== null` is more specific than we need, and potentially introduces bugs where it passes even though the value is potentially invalid e.g. an empty string.

It's better to just lean on Javascript coercion rules in these instances.

Reply:
Done

Reply:
done